apps/solidity-compiler/src/app/compiler-api.ts

```diff
@@ -249,11 +249,11 @@ export const CompilerApiMixin = (Base) => class extends Base {
     }
     this.on('fileManager', 'noFileSelected', this.data.eventHandlers.onNoFileSelected)
 
-    this.data.eventHandlers.onCompilationFinished = (success, data, source) => {
+    this.data.eventHandlers.onCompilationFinished = (success, data, source, input, version) => {
       this.compileErrors = data
       if (success) {
         // forwarding the event to the appManager infra
-        this.emit('compilationFinished', source.target, source, 'soljson', data)
+        this.emit('compilationFinished', source.target, source, 'soljson', data, input, version)
```

Comment:
updating the signature of `compilationResult` is a big changes.. 
i.e we can keep it internal, but there's a tons of places where we are using the previous event signature.

Reply:
But this should be backward compatible I think